src/main/java/org/jenkinsci/plugins/pluginusage/analyzer/BuildWrapperJobAnalyzer.java

```diff
@@ -25,11 +26,12 @@ public BuildWrapperJobAnalyzer() {
 			plugins.add(usedPlugin);
 		}
 	}
-	
-	protected void doJobAnalyze(AbstractProject item, HashMap<PluginWrapper, JobsPerPlugin> mapJobsPerPlugin)
+
+	@Override
+	protected void doJobAnalyze(Job item, Map<PluginWrapper, JobsPerPlugin> mapJobsPerPlugin)
 	{
 		super.doJobAnalyze(null, mapJobsPerPlugin);
-		if (item instanceof BuildableItemWithBuildWrappers)
+		if (item != null && item instanceof BuildableItemWithBuildWrappers)
```

Comment:
It is not needed to check for null. The instanceof will also work if item is null. 
null instanceof anything will be false.